### ISSUE TITLE
import-locality is not exist

### ISSUE THREAD

**Issue by meeof** [feature-sliced/steiger#142]
the option "import-locality" described in the documentation is not exist in the package

**Comment by illright** [comment]
Perhaps it's because you need to add `fsd/` before the rule name? Sorry for the confusion, we should update the readme

**Comment by meeof** [comment]
i tried this 
`Invalid configuration in steiger.config.js: Required at "[2].ignores", or Invalid enum value. Expected 'fsd/ambiguous-slice-names' | 'fsd/excessive-slicing' | 'fsd/forbidden-imports' | 'fsd/inconsisten
t-naming' | 'fsd/insignificant-slice' | 'fsd/no-layer-public-api' | 'fsd/no-public-api-sidestep' | 'fsd/no-reserved-folder-names' | 'fsd/no-segmentless-slices' | 'fsd/no-segments-on-sliced-layers' | 'f
sd/no-ui-in-app' | 'fsd/public-api' | 'fsd/repetitive-naming' | 'fsd/segments-by-purpose' | 'fsd/shared-lib-grouping' | 'fsd/typo-in-layer-name' | 'fsd/no-processes', received 'fsd/import-locality' at "[2].rules.fsd/import-locality", or Required at "[2].meta"; Required at "[2].ruleDefinitions"
`

**Comment by pestrige** [comment]
> Ah, sorry, I see what the problem is now. We forgot to include that rule in the ruleset. We'll add it in the next release, thanks for noticing!

Hello! Do you know when this issue will be fixed? At the moment, it is still not possible to use the `fsd/import-locality` rule.

**Comment by Solant** [comment]
> Hello! Do you know when this issue will be fixed? At the moment, it is still not possible to use the `fsd/import-locality` rule.

Yes, the issue is already resolved. But the rule is disabled by default, so you need to enable it in `steiger.config.ts`:

```typescript
import fsd from '@feature-sliced/steiger-plugin';
import { defineConfig } from 'steiger';

export default defineConfig([
	...fsd.configs.recommended,
	{
		files: ['./src/**'],
		rules: {
			'fsd/import-locality': 'error',
		},
	},
])
```
